File: IDE/MonoDevelop/MonoDevelop.MonoGame/templates/Common/Processor1.cs
using System;
using System.Collections.Generic;
using System.Linq;
using Microsoft.Xna.Framework;
using Microsoft.Xna.Framework.Graphics;
using Microsoft.Xna.Framework.Content.Pipeline;
using Microsoft.Xna.Framework.Content.Pipeline.Graphics;

using TInput = System.String;
using TOutput = System.String;

namespace ${Namespace}
{
	[ContentProcessor(DisplayName = "Processor1")]
	class Processor1 : ContentProcessor<TInput, TOutput>
	{
		public override TOutput Process(TInput input, ContentProcessorContext context)
		{
			return default(TOutput);
		}
	}
}

File: ProjectTemplates/DotNetTemplate/MonoGame.Templates.VisualBasic/content/PipelineExtension/Processor1.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports Microsoft.Xna.Framework
Imports Microsoft.Xna.Framework.Graphics
Imports Microsoft.Xna.Framework.Content.Pipeline
Imports Microsoft.Xna.Framework.Content.Pipeline.Graphics

Imports TInput = System.String
Imports TOutput = System.String

Namespace BlankApp
    <ContentProcessor(DisplayName:="Processor1")>
    Class Processor1 : Inherits ContentProcessor(Of TInput, TOutput)

        Public Overrides Function Process(ByVal input As TInput, ByVal context As ContentProcessorContext) As TOutput
            Return Nothing
        End Function
    End Class
End Namespace

